app/views/brexit_checker/show.html.erb

```diff
@@ -15,15 +15,15 @@
 
 <% content_for :breadcrumbs do %>
   <div class="govuk-width-container">
-    <% if @previous_page %>
-      <%= render "govuk_publishing_components/components/back_link", {
-        href: transition_checker_questions_path(page: @previous_page, c: criteria_keys),
-      } %>
-    <% else %>
-      <%= render "govuk_publishing_components/components/back_link", {
-        href: "/transition",
-      } %>
-    <% end %>
+    <%= render "govuk_publishing_components/components/back_link", {
+      href: @previous_page ? transition_checker_questions_path(page: @previous_page, c: criteria_keys) : "/transition",
```

Comment:
V minor style comment feel free to ignore:
You could use a variable (like is done for formatted_description above) to cut this line size down. eg
```
breadcrumb_path = "/transition"
if  @previous_page
  breadcrumb_path = blah
end
```